packages/run-protocol/test/vaultFactory/swingsetTests/governance/vat-alice.js

```diff
@@ -3,13 +3,27 @@
 import { E } from '@agoric/eventual-send';
 import { Far } from '@agoric/marshal';
 import { AmountMath } from '@agoric/ertp';
+import { daysForVoteToClose } from './bootstrap';
+import { ONE_DAY } from '../setup';
 
 const { details: X, quote: q } = assert;
 
+/**
+ *
+ * @param {(msg: any)=> void} log
+ * @param {ZoeService} zoe
+ * @param {Brand[]} brands
+ * @param {Payment[]} payments
+ * @param {ManualTimer} timer configured to tick one day (@see setup.js)
+ * @returns {Promise<{startTest: Function}>}
+ */
 const build = async (log, zoe, brands, payments, timer) => {
   const [moolaBrand] = brands;
   const [moolaPayment] = payments;
 
+  /**
+   * @param {vaultFactoryPublicFacet} vaultFactory
```

Comment:
Does the type name `vaultFactoryPublicFacet` really start with a lower-case `v`?

Reply:
Yep. If this was incorrect it would fail the type checker. But glad you mentioned it because I'll correct the capitalization of the typedef.

Reply:
Would now be a good time to start linting naming conventions? https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/naming-convention.md

Reply:
1. I think we already do enforce variable naming (vs. type naming)
 2. not "now" as in this PR, but a separate PR could work